### DISCUSSION TITLE
Azure join for identities across resource groups

### DISCUSSION THREAD

**Discussion by iMisteg** [gravitational/teleport#34481] [Q&A] [answered]
I encountered an issue on agent version 14.1.1 described in this merged [pull request](https://github.com/gravitational/teleport/pull/28927). If the identity and the virtual machine are in different resource groups, when attempting to start the agent on the virtual machine, I receive this error:
`ERRO [PROC:1]    Node failed to establish connection to cluster: rpc error: code = PermissionDenied desc = no VM found with matching VM ID. pid:3504.1 service/connect.go:123
`
I don't know what to do, please help.

**Comment by AleksanderTolstopyatov** [comment, marked as answer]
Problem solved. I didn't carefully read the bicep files from the documentation. They create an identity with access to only one RG
